checks.d/redisdb.py

```diff
@@ -329,3 +330,9 @@ def check(self, instance):
 
         self._check_db(instance, custom_tags)
         self._check_slowlog(instance, custom_tags)
+
+    def _collect_metadata(self, info):
+        metadata_dict = {}
+        if info and 'redis_version' in info:
+            metadata_dict['version'] = info['redis_version'].split('.')
```

Comment:
why not sending it as a string directly ? 
It probably better to have the same type for all metadata